### PR TITLE
Make the publish step have access to the aws secrets

### PR DESCRIPTION
**Pull request by JakeChampion**

### CODE REVIEW

File: .github/workflows/publish.yml [hunk 1]
```diff
@@ -22,3 +22,5 @@ jobs:
     - run: npm publish
       env:
         NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
+        AWS_ACCESS_KEY: ${{ secrets.AWS_ACCESS_KEY }}
+        AWS_SECRET_KEY: ${{ secrets.AWS_SECRET_KEY }}
```
Comment (notlee) [inline .github/workflows/publish.yml:26]:
```suggestion
```

Comment (JakeChampion) [reply .github/workflows/publish.yml:26]: We need this env var, it's not a duplicate of the one above 😅

Comment (notlee) [reply .github/workflows/publish.yml:26]: wow my eyes and or brain 😅

**Review by notlee — APPROVED** [review]

**Comment by origamiserviceuser** [comment]
:tada: This PR is included in version v1.5.3 :tada:

**Pull request merged**